mcs/class/System.ServiceModel/Test/System.ServiceModel.Description/ServiceEndpointTest.cs

```diff
@@ -57,13 +57,15 @@ public void NullArguments2 ()
 			new ServiceEndpoint (null, null, null);
 		}
 
+#if !MONOTOUCH_WATCH
 		[Test]
 		public void ListenUri ()
```

Comment:
Please change this to:

```csharp
#if FEATURE_NO_BSD_SOCKETS
	[ExpectedException (typeof (PlatformNotSupportedException))]
#endif
```

We're using this pattern for WatchOS socket code everywhere else instead of MONOTOUCH_WATCH.